packages/expo-constants/scripts/getAppConfig.js

```diff
@@ -15,6 +15,7 @@ if (fs.existsSync(path.join(possibleProjectRoot, 'package.json'))) {
 
 process.chdir(projectRoot);
 
+require('@expo/env').load(projectRoot);
```

Comment:
feel meh about this because we don't depend on `@expo/env` in expo-constants. should we add an option to `getConfig` that has it load `@expo/env` instead? eg: `loadEnv: true`